### PR TITLE
update ru_RU translation

### PR DESCRIPTION
**Pull request by dmitrygribenchuk**
# Description

Hello. Here is update for translation ru_RU

### CODE REVIEW

File: resources/i18n/ru_RU/cura.po [hunk 1]
```diff
@@ -476,14 +476,14 @@ msgstr "Фреймворк приложения"
...
+msgstr "Применить к"
 
 msgctxt "@label"
 msgid "Apply Extruder offsets to GCode"
-msgstr "Применить смещения экструдера к GCode"
+msgstr "Применить смещения экструдера к G-коду"
 
 msgctxt "@info:title"
-msgid "Are you ready for cloud printing?"
+msgid "Are you ready for cloud printing?"коду
```
Comment (wawanbreton) [inline resources/i18n/ru_RU/cura.po:486]: This seems to be a mistake

**Review by wawanbreton — COMMENTED** [review]
Just a remark for a change that seems to be a mistake, otherwise looks good !

File: resources/i18n/ru_RU/cura.po (hunk 1 above)
Comment (dmitrygribenchuk) [reply resources/i18n/ru_RU/cura.po]: My bad. Fixed

**Comment by wawanbreton** [comment]
Looks good now, many thanks for the review !

**Pull request merged**